semantic_model_generator/validate/fields.py

```diff
@@ -0,0 +1,13 @@
+from semantic_model_generator.protos.semantic_model_pb2 import Table
+
+def validate_contains_datatype_for_each_col(table: Table) -> None:
+    # Ensure every col for every table has 'data_type' present.
+    for dim_col in table.dimensions:
+        if dim_col.data_type is None or len(dim_col.data_type) < 2: # account for spaces
```

Comment:
Nit, we can do `.strip()` to remove spaces

Reply:
true!